spec/models/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+require "shoulda/matchers"
+
+RSpec.describe User, type: :model do
+  context "should have columns" do
+    it {is_expected.to have_db_column(:name).of_type(:string)}
+    it {is_expected.to have_db_column(:money).of_type(:float)}
+    it {is_expected.to have_db_column(:created_at).of_type(:datetime)}
+    it {is_expected.to have_db_column(:updated_at).of_type(:datetime)}
+    it {is_expected.to have_db_column(:email).of_type(:string)}
+    it {is_expected.to have_db_column(:admin).of_type :boolean}
+  end
+
+  context "associations" do
+    it {is_expected.to have_many(:notifications).dependent(:destroy)}
+    it {is_expected.to have_many(:score_bets).dependent(:destroy)}
+    it {is_expected.to have_many(:comments).dependent(:destroy)}
+  end
+
+  context "validate presence and length" do
+    it {is_expected.to validate_presence_of :name}
+    it {is_expected.to validate_length_of(:name).is_at_most(50)}
```

Comment:
check việc message cho hiển thị đúng ko nữa nhé
example `it {is_expected.to validate_presence_of(:name)
      .with_message(I18n.t("activerecord.errors.models.club.attributes.name.blank"))}`